src/custom/types/ethers.d.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Additional typings extensions for Ethers.js.
+ */
+
+import { ethers } from 'ethers'
+
+declare module 'ethers' {
+  /**
+   * A signature-like type.
+   */
+  export type SignatureLike = Parameters<typeof ethers.utils.splitSignature>[0]
+
+  /**
+   * EIP-712 typed data domain.
+   */
+  export type TypedDataDomain = Parameters<typeof ethers.utils._TypedDataEncoder.hashDomain>[0]
```

Comment:
I'm not sure it's a good idea to just create new exports
If we need them, declare them locally

You can get at least `SignatureLike`  from
```ts
import { SignatureLike } from '@ethersproject/bytes'